### ISSUE TITLE
[Bug] bottom URL/Navigation bar obscures bottom of page content, on a short website where there is no scrolling the hidden content cannot be viewed as there is no way to scroll to hide the URL bar

### ISSUE THREAD

**Issue by thomasstani** [mozilla-mobile/fenix#14991]
## Steps to reproduce

Go to a website that is relatively short: i.e. https://actionlauncher.com/about

Try to use the bottom site navigation on the website, which will be blocked by the bottom URL bar of Fenix.

### Expected behavior

URL bar should not obscure the bottom of the page

### Actual behavior

URL bar obscures the bottom of the page!

### Device information

Samsung Galaxy s9+,  android 10, one UI 2.1

Fenix 80.1.3 (Build #2015762835)

* Android device: ?
* Fenix version: ?

**Comment by Mugurell** [comment]
Hello,
Thank you for the report!

A video would help here but I think what you're seeing is that the page is also not scrollable?
Because it seems to fit almost entirely the display?
If so this is a duplicate of https://github.com/mozilla-mobile/fenix/issues/13797

**Comment by thomasstani** [comment]
Hi Mugurell,

Yes, my apologies, it does indeed seem to be a duplicate of #13797 , this should probably be a note on that ticket, not a new ticket.

 Best regards,
